src/applications/vaos/new-appointment/components/CommunityCareProviderSelectionPage/ProviderSelectionField.jsx

```diff
@@ -75,9 +75,11 @@ function ProviderSelectionField({
   useEffect(
     () => {
       if (showProvidersList) {
-        scrollAndFocus('h2');
-      } else if (mounted) {
+        scrollAndFocus('#providerSelectionHeader');
+      } else if (mounted && !providerSelected) {
         scrollAndFocus('.va-button-link');
+      } else if (mounted) {
+        scrollAndFocus('#selectedProvider');
```

Comment:
Now, we only focus on the button link if there's no provider selected. Otherwise, we focus on the selected provider region